src/common/options.cc

```diff
@@ -7881,6 +7881,26 @@ std::vector<Option> get_mds_options() {
      .set_default(0)
      .set_description("threshold for cache usage to disallow \"dump cache\" operation to file")
      .set_long_description("Disallow MDS from dumping caches to file via \"dump cache\" command if cache usage exceeds this threshold."),
+    
+    Option("mds_request_load_average_decay_rate", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(60)
+    .set_description("rate of decay in seconds for calculating request load average"),
+
+    Option("mds_worm_commit_period", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+     .set_default(120)
+     .set_description("default auto commit period for worm feature"),
+
+    Option("mds_worm_retention_period", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
Seems these three options are all sharing the same default value. Did you do that deliberately?

Also I've observed you silently changed the perm of all files you touched from 100644->100755, which I guess should definitely need to be addressed first..

Reply:
These options had been redesigned， I think `mds_worm_retention_period`  may be equal to `mds_worm_min_retention_period`  and set them to  "1h",  `mds_worm_max_retention_period`  may be larger than 'mds_worm_min_retention_period` and set it to "30 years", What do you think?

the modified file mode already restore to 100644,  thanks!

Reply:
You might switch to the inline .set_min & .set_max helpers to perform the boundary checking, then

Reply:
fixed it，thanks